sites/footers/includes/footer.html

```diff
@@ -1,65 +1,130 @@
+				<li><a href="https://travel.gc.ca/">Travel and tourism</a></li>
+				<li><a href="https://www.canada.ca/en/services/business.html">Business and industry</a></li>
+				<li><a href="https://www.canada.ca/en/services/benefits.html">Benefits</a></li>
+				<li><a href="https://www.canada.ca/en/services/health.html">Health</a></li>
+				<li><a href="https://www.canada.ca/en/services/taxes.html">Taxes</a></li>
+				<li><a href="https://www.canada.ca/en/services/environment.html">Environment and natural resources</a></li>
+				<li><a href="https://www.canada.ca/en/services/defence.html">National security and defence</a></li>
+				<li><a href="https://www.canada.ca/en/services/culture.html">Culture, history and sport</a></li>
+				<li><a href="https://www.canada.ca/en/services/policing.html">Policing, justice and emergencies</a></li>
+				<li><a href="https://www.canada.ca/en/services/transport.html">Transport and infrastructure</a></li>
+				<li><a href="https://international.gc.ca/world-monde/index.aspx?lang=eng">Canada and the world</a></li>
+				<li><a href="https://www.canada.ca/en/services/finance.html">Money and finances</a></li>
+				<li><a href="https://www.canada.ca/en/services/science.html">Science and innovation</a></li>
+				<li><a href="https://www.canada.ca/en/services/indigenous-peoples.html">Indigenous peoples</a></li>
+				<li><a href="https://www.canada.ca/en/services/veterans.html">Veterans</a></li>
+				<li><a href="https://www.canada.ca/en/services/youth.html">Youth</a></li>
 {%- endif -%}
 			</ul>
 		</nav>
 	</div>
 {% endunless %}
 	<div class="brand">
 	<div class="container">
-		<div class="row ">
-{% unless page.noFooterSite %}
-			<nav class="col-md-9 col-lg-10 ftr-urlt-lnk">
-				<h2 class="wb-inv">{{ i18nText-footerSite }}</h2>
-				<ul>
-{%- if i18nText-lang == "fr" -%}
+		<div class="brand-row">
+			<nav class="ftr-urlt-lnk">
+				<h3 class="wb-inv">{{ i18nText-supportLinks }}</h3>
+				{% unless page.noFooterSite %}
+				<ul class="spprt-lnks">
+					{%- if i18nText-lang == "fr" -%}
 					<li><a href="https://www.canada.ca/fr/sociaux.html">Médias sociaux</a></li>
 					<li><a href="https://www.canada.ca/fr/mobile.html">Applications mobiles</a></li>
 					<li><a href="https://www1.canada.ca/fr/nouveausite.html">À propos de Canada.ca</a></li>
-					<li><a href="https://www.canada.ca/fr/transparence/avis.html">Avis</a></li>
-					<li><a href="https://www.canada.ca/fr/transparence/confidentialite.html">Confidentialité</a></li>
-{%- elsif i18nText-lang == "en" -%}
+					{%- elsif i18nText-lang == "en" -%}
 					<li><a href="https://www.canada.ca/en/social.html">Social media</a></li>
 					<li><a href="https://www.canada.ca/en/mobile.html">Mobile applications</a></li>
 					<li><a href="https://www1.canada.ca/en/newsite.html">About Canada.ca</a></li>
+					{%- endif -%}
+				</ul>
+				{% endunless %}
+				<h4 class="wb-inv">{{ i18nText-transparency }}</h4>
+				<ul class="trnsprncy-lnks">
```

Comment:
I didn't found specific style associated to it.

```suggestion
				<ul>
```

Reply:
You are right. I had some styles associated to it, but after cleaning up the code, it was no longer necessary. I'll remove it.